modules/conversion-center/admin-menu-items/conversion-center-menu-item.php

```diff
@@ -10,7 +10,7 @@
 
 class Conversion_Center_Menu_Item implements Admin_Menu_Item, Admin_Menu_Item_Has_Position {
 
-	const AFTER_ELEMENTOR = 58.7;
+	const AFTER_ELEMENTOR = '58.7';
```

Comment:
Why?

<img width="466" alt="Screenshot 2024-05-08 at 9 58 14 AM" src="https://github.com/elementor/elementor/assets/20996103/084eb91b-8cb2-43e1-9b56-875d4136d0d2">

Reply:
@louiswol94 I used a string because when I use the constant elsewhere ( to add the class ) internally it 's used as a string. I will go back to a float and cast it to string elsewhere.